package/base-files/files/lib/upgrade/fwtool.sh

```diff
@@ -1,8 +1,10 @@
+CHECK_SYSUPGRADE_SIGNATURE="${CHECK_SYSUPGRADE_SIGNATURE:-$(uci get system.@system[-1].check_sysupgrade_signature)}"
+
 fwtool_check_signature() {
 	[ $# -gt 1 ] && return 1
 
 	[ ! -x /usr/bin/ucert ] && {
```

Comment:
This PR makes ucert mandatory, so I'm wondering if we can't simply drop this check and simplify the code little bit.

Reply:
If that's an option for you I'm happy to remove the code.

Reply:
But you've removed it completely now, it will always check the signature.

Reply:
in other words, there could be just oneliner `[ "$REQUIRE_IMAGE_SIGNATURE" = 1 ] || return ...`